__tests__/app.test.js

```diff
@@ -209,7 +212,54 @@ describe("POST /api/articles/:article_id/comments", () => {
       .send(newComment)
       .expect(404)
       .then(({ body }) => {
-        expect(body).toHaveProperty('msg', 'Not Found');
+        expect(body.msg).toBe('Not Found');
       });
   });
 });
+
+describe('PATCH /api/articles/:article_id', () => {
+  test('PATCH: 200 should update the article votes when given a number which will return an updated article', () => {
+    return request(app)
+    .patch("/api/articles/1")
+    .send ({ inc_votes:1 })
+    .expect(200)
+    .then(({ body }) => {
+      const { updatedArticle } = body;
+      expect(updatedArticle).toMatchObject({
+        article_id: 1,
+        votes: 101
+    })
+  })
+})
+test('PATCH: 200 should update the article votes when given a number which will return an updated article', () => {
+  return request(app)
+  .patch("/api/articles/1")
+  .send ({ inc_votes:-100 })
+  .expect(200)
+  .then(({ body }) => {
+    const { updatedArticle } = body;
+    expect(updatedArticle).toMatchObject({
+      article_id: 1,
+      votes: 0
+  })
+})
```

Comment:
test 200 when passing a body with extra properties
e.g. `.send({inc_votes:1, extraProperty:"should be ignored"})`